input/examples/hearing-threshold-1.xml

```diff
@@ -28,10 +28,7 @@
   </subject>
   <effectiveDateTime value="2020-11-06T10:45:00-05:00"/>
   <performer>
-    <reference value="PractitionerRole/audiologist-bradley-jill"/>
-  </performer>
-  <performer>
-    <reference value="Organization/adv-hearing-care"/>
```

Comment:
Removed from ig.xml but still need to delete the file. Organization-adv-hearing-care is still being picked up automatically in the IG.

Reply:
Done